config/initializers/darjeelink.rb

```diff
@@ -5,29 +5,31 @@
   config.domain = ENV['DOMAIN']
 
   config.source_mediums = {
+    chatbot: 'Chatbot',
```

Comment:
Ordered all options alphabetically 
Apologies for refactor at same time as feature change, but the Github diff is correctly differentiating what's just moved up/down and what's change

Reply:
Cant think of much to add here. If we do not sort it automatically, then maybe worth adding a comment to explain if people add to it we want to keep it lexicographically sorted?

Reply:
Good suggestion 🤔 
idk if sorting just before rendering is smarter or unnecessary 🤷 
I quite like the convention that all collections in code (as in array/hash/object literals) are sorted alphabetically by default, unless there's goo reason not too. Make it easier to read imo.

I'll a comment in an attempt to preserve future ordering - hopefully subsequent PRs to this area can keep this in line too